src/Nest/Cluster/RemoteInfo/ElasticClient-RemoteInfo.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Threading.Tasks;
+using Elasticsearch.Net;
+using System.Threading;
+
+namespace Nest
+{
+	public partial interface IElasticClient
+	{
+		/// <summary>
+		/// The cluster health API allows to get a very simple status on the health of the cluster.
```

Comment:
[I think this should use the remote info API summary?](https://github.com/elastic/elasticsearch/blob/f22e0dc30b629b4c1c5136f974d8dbc19d380420/docs/reference/cluster/remote-info.asciidoc)

> The cluster remote info API allows to retrieve all of the configured remote cluster information.